src/utils/script.ts

```diff
@@ -22,13 +22,13 @@ export const parseScriptData = (script: CollectionEntry<"scripts">): Script => {
       const entry = characterRegistry[parsedCharacterName];
 
       parsedCharacter = CharacterSchema.parse({
-        scriptType,
+        scriptType: "official",
         name: parsedCharacterName,
         team: entry.type,
       });
     } else {
       parsedCharacter = CharacterSchema.parse({
-        scriptType,
+        scriptType: "homebrew",
```

Comment:
The buffets aren't recognized as a homebrew script because they don't have an `almanac`. We can handle this better in the future but this solution is fine for now